test/SILOptimizer/definite_init_nsmanagedvalue.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -I %S/../ClangImporter/Inputs/custom-modules %s -emit-sil -enable-sil-ownership
+
+// REQUIRES: objc_interop
+
+// Make sure if we have an NSManagedObject without stored properties that DI
+// does not crash or emit an error.
+
+import Foundation
+import CoreData
+
+class Person : NSManagedObject {
+  enum MyError : Error {
+  case error
```

Comment:
Indentation?

Reply:
Sure. I will do it in a follow on commit.